exporter.py

```diff
@@ -54,12 +55,41 @@ def parse_params():
 
 def run_prometheus_server(port, collectors, *args):
     start_http_server(int(port))
+    collector_list = list()
     for c in collectors:
         REGISTRY.register(c)
+        collector_list.append(REGISTRY._get_names(c))
```

Comment:
is there another way but using the internal function? How about using collectors and just skipping the CollectorUp one?

Reply:
I check whether the collectors that are registered also return metrics. I have not found a better way to get names of timeseries the collector produces. But okay, I'll find another way.

Reply:
Usually we should have the names, since we are having $collectors here already.